diff --git a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute.php b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute.php
index 6697084389..f8853cfb4e 100644
--- a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute.php
+++ b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Catalog_Product_Attribute extends Mage_Adminhtml_Bloc
     public function __construct()
     {
         $this->_controller = 'catalog_product_attribute';
-        $this->_headerText = Mage::helper('catalog')->__('Manage Attributes');
+        $this->_headerText = Mage::helper('catalog')->__('Attributes');
         $this->_addButtonLabel = Mage::helper('catalog')->__('Add New Attribute');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Edit/Tabs.php b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Edit/Tabs.php
index e156dfe08f..6773c9c544 100644
--- a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Edit/Tabs.php
+++ b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Edit/Tabs.php
@@ -41,8 +41,8 @@ class Mage_Adminhtml_Block_Catalog_Product_Attribute_Edit_Tabs extends Mage_Admi
         $model = Mage::registry('entity_attribute');
 
         $this->addTab('labels', [
-            'label'     => Mage::helper('catalog')->__('Manage Label / Options'),
-            'title'     => Mage::helper('catalog')->__('Manage Label / Options'),
+            'label'     => Mage::helper('catalog')->__('Label / Options'),
+            'title'     => Mage::helper('catalog')->__('Label / Options'),
             'content'   => $this->getLayout()->createBlock('adminhtml/catalog_product_attribute_edit_tab_options')->toHtml(),
         ]);
 
diff --git a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Set/Toolbar/Main.php b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Set/Toolbar/Main.php
index 075c543dd5..68f8caf094 100644
--- a/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Set/Toolbar/Main.php
+++ b/app/code/core/Mage/Adminhtml/Block/Catalog/Product/Attribute/Set/Toolbar/Main.php
@@ -57,7 +57,7 @@ class Mage_Adminhtml_Block_Catalog_Product_Attribute_Set_Toolbar_Main extends Ma
      */
     protected function _getHeader()
     {
-        return Mage::helper('catalog')->__('Manage Attribute Sets');
+        return Mage::helper('catalog')->__('Attribute Sets');
     }
 
     /**
diff --git a/app/code/core/Mage/Adminhtml/Block/Checkout/Agreement.php b/app/code/core/Mage/Adminhtml/Block/Checkout/Agreement.php
index 7263449b91..ca47b9378d 100644
--- a/app/code/core/Mage/Adminhtml/Block/Checkout/Agreement.php
+++ b/app/code/core/Mage/Adminhtml/Block/Checkout/Agreement.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Checkout_Agreement extends Mage_Adminhtml_Block_Widge
     public function __construct()
     {
         $this->_controller      = 'checkout_agreement';
-        $this->_headerText      = Mage::helper('checkout')->__('Manage Terms and Conditions');
+        $this->_headerText      = Mage::helper('checkout')->__('Terms and Conditions');
         $this->_addButtonLabel  = Mage::helper('checkout')->__('Add New Condition');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/Cms/Page.php b/app/code/core/Mage/Adminhtml/Block/Cms/Page.php
index fa74a0c56e..246d99e317 100644
--- a/app/code/core/Mage/Adminhtml/Block/Cms/Page.php
+++ b/app/code/core/Mage/Adminhtml/Block/Cms/Page.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Cms_Page extends Mage_Adminhtml_Block_Widget_Grid_Con
     public function __construct()
     {
         $this->_controller = 'cms_page';
-        $this->_headerText = Mage::helper('cms')->__('Manage Pages');
+        $this->_headerText = Mage::helper('cms')->__('Pages');
 
         parent::__construct();
 
diff --git a/app/code/core/Mage/Adminhtml/Block/Customer.php b/app/code/core/Mage/Adminhtml/Block/Customer.php
index d1137b9992..ff4805f6ca 100644
--- a/app/code/core/Mage/Adminhtml/Block/Customer.php
+++ b/app/code/core/Mage/Adminhtml/Block/Customer.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Customer extends Mage_Adminhtml_Block_Widget_Grid_Con
     public function __construct()
     {
         $this->_controller = 'customer';
-        $this->_headerText = Mage::helper('customer')->__('Manage Customers');
+        $this->_headerText = Mage::helper('customer')->__('Customers');
         $this->_addButtonLabel = Mage::helper('customer')->__('Add New Customer');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/Promo/Quote/Edit/Tab/Coupons.php b/app/code/core/Mage/Adminhtml/Block/Promo/Quote/Edit/Tab/Coupons.php
index 55b830994f..11697ab6ae 100644
--- a/app/code/core/Mage/Adminhtml/Block/Promo/Quote/Edit/Tab/Coupons.php
+++ b/app/code/core/Mage/Adminhtml/Block/Promo/Quote/Edit/Tab/Coupons.php
@@ -28,7 +28,7 @@ class Mage_Adminhtml_Block_Promo_Quote_Edit_Tab_Coupons extends Mage_Adminhtml_B
      */
     public function getTabLabel()
     {
-        return Mage::helper('salesrule')->__('Manage Coupon Codes');
+        return Mage::helper('salesrule')->__('Coupon Codes');
     }
 
     /**
@@ -38,7 +38,7 @@ class Mage_Adminhtml_Block_Promo_Quote_Edit_Tab_Coupons extends Mage_Adminhtml_B
      */
     public function getTabTitle()
     {
-        return Mage::helper('salesrule')->__('Manage Coupon Codes');
+        return Mage::helper('salesrule')->__('Coupon Codes');
     }
 
     /**
diff --git a/app/code/core/Mage/Adminhtml/Block/Rating/Rating.php b/app/code/core/Mage/Adminhtml/Block/Rating/Rating.php
index 79356ef8f9..201e58d35c 100644
--- a/app/code/core/Mage/Adminhtml/Block/Rating/Rating.php
+++ b/app/code/core/Mage/Adminhtml/Block/Rating/Rating.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Rating_Rating extends Mage_Adminhtml_Block_Widget_Gri
     public function __construct()
     {
         $this->_controller = 'rating';
-        $this->_headerText = Mage::helper('rating')->__('Manage Ratings');
+        $this->_headerText = Mage::helper('rating')->__('Ratings');
         $this->_addButtonLabel = Mage::helper('rating')->__('Add New Rating');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/System/Currency.php b/app/code/core/Mage/Adminhtml/Block/System/Currency.php
index 3f89f8377a..581f78892e 100644
--- a/app/code/core/Mage/Adminhtml/Block/System/Currency.php
+++ b/app/code/core/Mage/Adminhtml/Block/System/Currency.php
@@ -73,7 +73,7 @@ class Mage_Adminhtml_Block_System_Currency extends Mage_Adminhtml_Block_Template
 
     protected function getHeader()
     {
-        return Mage::helper('adminhtml')->__('Manage Currency Rates');
+        return Mage::helper('adminhtml')->__('Currency Rates');
     }
 
     protected function getSaveButtonHtml()
diff --git a/app/code/core/Mage/Adminhtml/Block/System/Store/Store.php b/app/code/core/Mage/Adminhtml/Block/System/Store/Store.php
index 32d664111d..2d6f5bb9ee 100644
--- a/app/code/core/Mage/Adminhtml/Block/System/Store/Store.php
+++ b/app/code/core/Mage/Adminhtml/Block/System/Store/Store.php
@@ -25,7 +25,7 @@ class Mage_Adminhtml_Block_System_Store_Store extends Mage_Adminhtml_Block_Widge
     public function __construct()
     {
         $this->_controller  = 'system_store';
-        $this->_headerText  = Mage::helper('adminhtml')->__('Manage Stores');
+        $this->_headerText  = Mage::helper('adminhtml')->__('Stores');
         $this->setTemplate('system/store/container.phtml');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/Tag/Tag.php b/app/code/core/Mage/Adminhtml/Block/Tag/Tag.php
index ffb4a22377..91c4564f69 100644
--- a/app/code/core/Mage/Adminhtml/Block/Tag/Tag.php
+++ b/app/code/core/Mage/Adminhtml/Block/Tag/Tag.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Tag_Tag extends Mage_Adminhtml_Block_Widget_Grid_Cont
     public function __construct()
     {
         $this->_controller = 'tag_tag';
-        $this->_headerText = Mage::helper('tag')->__('Manage Tags');
+        $this->_headerText = Mage::helper('tag')->__('Tags');
         $this->_addButtonLabel = Mage::helper('tag')->__('Add New Tag');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Block/Tax/Rule.php b/app/code/core/Mage/Adminhtml/Block/Tax/Rule.php
index 4379814d1d..5ec0951256 100644
--- a/app/code/core/Mage/Adminhtml/Block/Tax/Rule.php
+++ b/app/code/core/Mage/Adminhtml/Block/Tax/Rule.php
@@ -24,7 +24,7 @@ class Mage_Adminhtml_Block_Tax_Rule extends Mage_Adminhtml_Block_Widget_Grid_Con
     public function __construct()
     {
         $this->_controller      = 'tax_rule';
-        $this->_headerText      = Mage::helper('tax')->__('Manage Tax Rules');
+        $this->_headerText      = Mage::helper('tax')->__('Tax Rules');
         $this->_addButtonLabel  = Mage::helper('tax')->__('Add New Tax Rule');
         parent::__construct();
     }
diff --git a/app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php b/app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php
index e8727e2b36..3a4cf44c93 100644
--- a/app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php
+++ b/app/code/core/Mage/Adminhtml/Helper/Help/Mapping.php
@@ -125,7 +125,7 @@ abstract class Mage_Adminhtml_Helper_Help_Mapping extends Mage_Core_Helper_Abstr
             /* System → Import/Export */
             'system_convert_gui' => 'store-operations/dataflow.html',
             'system_convert_profile' => 'store-operations/dataflow.html',
-            /* System → Manage Currency */
+            /* System → Currency */
             'system_currency' => 'store-operations/currency-rates.html',
             'system_currencysymbol' => 'store-operations/currency-symbols.html',
             /* System → Transactional emails */
diff --git a/app/code/core/Mage/Adminhtml/controllers/Catalog/CategoryController.php b/app/code/core/Mage/Adminhtml/controllers/Catalog/CategoryController.php
index ad534a4be6..36f87fc1c3 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Catalog/CategoryController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Catalog/CategoryController.php
@@ -37,8 +37,7 @@ class Mage_Adminhtml_Catalog_CategoryController extends Mage_Adminhtml_Controlle
     protected function _initCategory($getRootInstead = false)
     {
         $this->_title($this->__('Catalog'))
-             ->_title($this->__('Categories'))
-             ->_title($this->__('Manage Categories'));
+             ->_title($this->__('Categories'));
 
         $categoryId = (int) $this->getRequest()->getParam('id', false);
         $storeId    = (int) $this->getRequest()->getParam('store');
@@ -193,8 +192,8 @@ class Mage_Adminhtml_Catalog_CategoryController extends Mage_Adminhtml_Controlle
             ->setContainerCssClass('catalog-categories');
 
         $this->_addBreadcrumb(
-            Mage::helper('catalog')->__('Manage Catalog Categories'),
-            Mage::helper('catalog')->__('Manage Categories')
+            Mage::helper('catalog')->__('Catalog Categories'),
+            Mage::helper('catalog')->__('Categories')
         );
 
         $block = $this->getLayout()->getBlock('catalog.wysiwyg.js');
diff --git a/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/AttributeController.php b/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/AttributeController.php
index b0e6e1f717..997162cc29 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/AttributeController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/AttributeController.php
@@ -56,7 +56,7 @@ class Mage_Adminhtml_Catalog_Product_AttributeController extends Mage_Adminhtml_
     {
         $this->_title($this->__('Catalog'))
              ->_title($this->__('Attributes'))
-             ->_title($this->__('Manage Attributes'));
+             ->_title($this->__('Attributes'));
 
         if ($this->getRequest()->getParam('popup')) {
             $this->loadLayout('popup');
@@ -65,8 +65,8 @@ class Mage_Adminhtml_Catalog_Product_AttributeController extends Mage_Adminhtml_
                 ->_setActiveMenu('catalog/attributes/attributes')
                 ->_addBreadcrumb(Mage::helper('catalog')->__('Catalog'), Mage::helper('catalog')->__('Catalog'))
                 ->_addBreadcrumb(
-                    Mage::helper('catalog')->__('Manage Product Attributes'),
-                    Mage::helper('catalog')->__('Manage Product Attributes')
+                    Mage::helper('catalog')->__('Product Attributes'),
+                    Mage::helper('catalog')->__('Product Attributes')
                 )
             ;
         }
diff --git a/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/SetController.php b/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/SetController.php
index 9346139661..b5f6c174ac 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/SetController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Catalog/Product/SetController.php
@@ -31,7 +31,7 @@ class Mage_Adminhtml_Catalog_Product_SetController extends Mage_Adminhtml_Contro
     {
         $this->_title($this->__('Catalog'))
              ->_title($this->__('Attributes'))
-             ->_title($this->__('Manage Attribute Sets'));
+             ->_title($this->__('Attribute Sets'));
 
         $this->_setTypeId();
 
@@ -40,8 +40,8 @@ class Mage_Adminhtml_Catalog_Product_SetController extends Mage_Adminhtml_Contro
 
         $this->_addBreadcrumb(Mage::helper('catalog')->__('Catalog'), Mage::helper('catalog')->__('Catalog'));
         $this->_addBreadcrumb(
-            Mage::helper('catalog')->__('Manage Attribute Sets'),
-            Mage::helper('catalog')->__('Manage Attribute Sets')
+            Mage::helper('catalog')->__('Attribute Sets'),
+            Mage::helper('catalog')->__('Attribute Sets')
         );
 
         $this->_addContent($this->getLayout()->createBlock('adminhtml/catalog_product_attribute_set_toolbar_main'));
@@ -54,7 +54,7 @@ class Mage_Adminhtml_Catalog_Product_SetController extends Mage_Adminhtml_Contro
     {
         $this->_title($this->__('Catalog'))
              ->_title($this->__('Attributes'))
-             ->_title($this->__('Manage Attribute Sets'));
+             ->_title($this->__('Attribute Sets'));
 
         $this->_setTypeId();
         $attributeSet = Mage::getModel('eav/entity_attribute_set')
@@ -75,8 +75,8 @@ class Mage_Adminhtml_Catalog_Product_SetController extends Mage_Adminhtml_Contro
 
         $this->_addBreadcrumb(Mage::helper('catalog')->__('Catalog'), Mage::helper('catalog')->__('Catalog'));
         $this->_addBreadcrumb(
-            Mage::helper('catalog')->__('Manage Product Sets'),
-            Mage::helper('catalog')->__('Manage Product Sets')
+            Mage::helper('catalog')->__('Product Sets'),
+            Mage::helper('catalog')->__('Product Sets')
         );
 
         $this->_addContent($this->getLayout()->createBlock('adminhtml/catalog_product_attribute_set_main'));
@@ -177,7 +177,7 @@ class Mage_Adminhtml_Catalog_Product_SetController extends Mage_Adminhtml_Contro
     {
         $this->_title($this->__('Catalog'))
              ->_title($this->__('Attributes'))
-             ->_title($this->__('Manage Attribute Sets'))
+             ->_title($this->__('Attribute Sets'))
              ->_title($this->__('New Set'));
 
         $this->_setTypeId();
diff --git a/app/code/core/Mage/Adminhtml/controllers/Catalog/ProductController.php b/app/code/core/Mage/Adminhtml/controllers/Catalog/ProductController.php
index a10f7e2e71..a41cc758bd 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Catalog/ProductController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Catalog/ProductController.php
@@ -65,7 +65,7 @@ class Mage_Adminhtml_Catalog_ProductController extends Mage_Adminhtml_Controller
     protected function _initProduct()
     {
         $this->_title($this->__('Catalog'))
-             ->_title($this->__('Manage Products'));
+             ->_title($this->__('Products'));
 
         $productId  = (int) $this->getRequest()->getParam('id');
         $product    = Mage::getModel('catalog/product')
@@ -184,7 +184,7 @@ class Mage_Adminhtml_Catalog_ProductController extends Mage_Adminhtml_Controller
     public function indexAction()
     {
         $this->_title($this->__('Catalog'))
-             ->_title($this->__('Manage Products'));
+             ->_title($this->__('Products'));
 
         $this->loadLayout();
         $this->renderLayout();
diff --git a/app/code/core/Mage/Adminhtml/controllers/Cms/PageController.php b/app/code/core/Mage/Adminhtml/controllers/Cms/PageController.php
index 9924aa8998..c7dd285d5f 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Cms/PageController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Cms/PageController.php
@@ -32,7 +32,7 @@ class Mage_Adminhtml_Cms_PageController extends Mage_Adminhtml_Controller_Action
         $this->loadLayout()
             ->_setActiveMenu('cms/page')
             ->_addBreadcrumb(Mage::helper('cms')->__('CMS'), Mage::helper('cms')->__('CMS'))
-            ->_addBreadcrumb(Mage::helper('cms')->__('Manage Pages'), Mage::helper('cms')->__('Manage Pages'))
+            ->_addBreadcrumb(Mage::helper('cms')->__('Pages'), Mage::helper('cms')->__('Pages'))
         ;
         return $this;
     }
@@ -44,7 +44,7 @@ class Mage_Adminhtml_Cms_PageController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('CMS'))
              ->_title($this->__('Pages'))
-             ->_title($this->__('Manage Content'));
+             ->_title($this->__('Content'));
 
         $this->_initAction();
         $this->renderLayout();
@@ -66,7 +66,7 @@ class Mage_Adminhtml_Cms_PageController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('CMS'))
              ->_title($this->__('Pages'))
-             ->_title($this->__('Manage Content'));
+             ->_title($this->__('Content'));
 
         // 1. Get ID and create model
         $id = $this->getRequest()->getParam('page_id');
diff --git a/app/code/core/Mage/Adminhtml/controllers/CustomerController.php b/app/code/core/Mage/Adminhtml/controllers/CustomerController.php
index 4cc278cdcd..dfc9c2a998 100644
--- a/app/code/core/Mage/Adminhtml/controllers/CustomerController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/CustomerController.php
@@ -45,7 +45,7 @@ class Mage_Adminhtml_CustomerController extends Mage_Adminhtml_Controller_Action
      */
     protected function _initCustomer($idFieldName = 'id')
     {
-        $this->_title($this->__('Customers'))->_title($this->__('Manage Customers'));
+        $this->_title($this->__('Customers'))->_title($this->__('Customers'));
 
         $customerId = (int) $this->getRequest()->getParam($idFieldName);
         $customer = Mage::getModel('customer/customer');
@@ -63,7 +63,7 @@ class Mage_Adminhtml_CustomerController extends Mage_Adminhtml_Controller_Action
      */
     public function indexAction()
     {
-        $this->_title($this->__('Customers'))->_title($this->__('Manage Customers'));
+        $this->_title($this->__('Customers'))->_title($this->__('Customers'));
 
         if ($this->getRequest()->getQuery('ajax')) {
             $this->_forward('grid');
@@ -87,7 +87,6 @@ class Mage_Adminhtml_CustomerController extends Mage_Adminhtml_Controller_Action
          * Add breadcrumb item
          */
         $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Customers'), Mage::helper('adminhtml')->__('Customers'));
-        $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Manage Customers'), Mage::helper('adminhtml')->__('Manage Customers'));
 
         $this->renderLayout();
     }
diff --git a/app/code/core/Mage/Adminhtml/controllers/RatingController.php b/app/code/core/Mage/Adminhtml/controllers/RatingController.php
index 08a1d5fa62..9d63942881 100644
--- a/app/code/core/Mage/Adminhtml/controllers/RatingController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/RatingController.php
@@ -33,7 +33,7 @@ class Mage_Adminhtml_RatingController extends Mage_Adminhtml_Controller_Action
         $this->loadLayout();
 
         $this->_setActiveMenu('catalog/reviews_ratings/ratings');
-        $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Manage Ratings'), Mage::helper('adminhtml')->__('Manage Ratings'));
+        $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Ratings'), Mage::helper('adminhtml')->__('Ratings'));
         $this->_addContent($this->getLayout()->createBlock('adminhtml/rating_rating'));
 
         $this->renderLayout();
@@ -52,7 +52,7 @@ class Mage_Adminhtml_RatingController extends Mage_Adminhtml_Controller_Action
         $this->_title($ratingModel->getId() ? $ratingModel->getRatingCode() : $this->__('New Rating'));
 
         $this->_setActiveMenu('catalog/reviews_ratings/ratings');
-        $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Manage Ratings'), Mage::helper('adminhtml')->__('Manage Ratings'));
+        $this->_addBreadcrumb(Mage::helper('adminhtml')->__('Ratings'), Mage::helper('adminhtml')->__('Ratings'));
 
         $this->_addContent($this->getLayout()->createBlock('adminhtml/rating_edit'))
             ->_addLeft($this->getLayout()->createBlock('adminhtml/rating_edit_tabs'));
@@ -142,7 +142,7 @@ class Mage_Adminhtml_RatingController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Catalog'))
              ->_title($this->__('Reviews and Ratings'))
-             ->_title($this->__('Manage Ratings'));
+             ->_title($this->__('Ratings'));
 
         Mage::register('entityId', Mage::getModel('rating/rating_entity')->getIdByCode('product'));
     }
diff --git a/app/code/core/Mage/Adminhtml/controllers/System/CurrencyController.php b/app/code/core/Mage/Adminhtml/controllers/System/CurrencyController.php
index 2a042a75a5..700e1df719 100644
--- a/app/code/core/Mage/Adminhtml/controllers/System/CurrencyController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/System/CurrencyController.php
@@ -47,7 +47,7 @@ class Mage_Adminhtml_System_CurrencyController extends Mage_Adminhtml_Controller
      */
     public function indexAction()
     {
-        $this->_title($this->__('System'))->_title($this->__('Manage Currency Rates'));
+        $this->_title($this->__('System'))->_title($this->__('Currency Rates'));
 
         $this->loadLayout();
         $this->_setActiveMenu('system/currency');
diff --git a/app/code/core/Mage/Adminhtml/controllers/System/StoreController.php b/app/code/core/Mage/Adminhtml/controllers/System/StoreController.php
index 87154be97c..f98b37bc6c 100644
--- a/app/code/core/Mage/Adminhtml/controllers/System/StoreController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/System/StoreController.php
@@ -49,7 +49,7 @@ class Mage_Adminhtml_System_StoreController extends Mage_Adminhtml_Controller_Ac
         $this->loadLayout()
             ->_setActiveMenu('system/store')
             ->_addBreadcrumb(Mage::helper('adminhtml')->__('System'), Mage::helper('adminhtml')->__('System'))
-            ->_addBreadcrumb(Mage::helper('adminhtml')->__('Manage Stores'), Mage::helper('adminhtml')->__('Manage Stores'))
+            ->_addBreadcrumb(Mage::helper('adminhtml')->__('Stores'), Mage::helper('adminhtml')->__('Stores'))
         ;
         return $this;
     }
diff --git a/app/code/core/Mage/Adminhtml/controllers/Tax/Class/CustomerController.php b/app/code/core/Mage/Adminhtml/controllers/Tax/Class/CustomerController.php
index d86f3e6bc6..252cf41d21 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Tax/Class/CustomerController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Tax/Class/CustomerController.php
@@ -160,7 +160,7 @@ class Mage_Adminhtml_Tax_Class_CustomerController extends Mage_Adminhtml_Control
             ->_setActiveMenu('sales/tax/classes_customer')
             ->_addBreadcrumb(Mage::helper('tax')->__('Sales'), Mage::helper('tax')->__('Sales'))
             ->_addBreadcrumb(Mage::helper('tax')->__('Tax'), Mage::helper('tax')->__('Tax'))
-            ->_addBreadcrumb(Mage::helper('tax')->__('Manage Customer Tax Classes'), Mage::helper('tax')->__('Manage Customer Tax Classes'))
+            ->_addBreadcrumb(Mage::helper('tax')->__('Customer Tax Classes'), Mage::helper('tax')->__('Customer Tax Classes'))
         ;
         return $this;
     }
diff --git a/app/code/core/Mage/Adminhtml/controllers/Tax/Class/ProductController.php b/app/code/core/Mage/Adminhtml/controllers/Tax/Class/ProductController.php
index ffdada2720..9e4c47e770 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Tax/Class/ProductController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Tax/Class/ProductController.php
@@ -159,7 +159,7 @@ class Mage_Adminhtml_Tax_Class_ProductController extends Mage_Adminhtml_Controll
             ->_setActiveMenu('sales/tax/classes_product')
             ->_addBreadcrumb(Mage::helper('tax')->__('Sales'), Mage::helper('tax')->__('Sales'))
             ->_addBreadcrumb(Mage::helper('tax')->__('Tax'), Mage::helper('tax')->__('Tax'))
-            ->_addBreadcrumb(Mage::helper('tax')->__('Manage Product Tax Classes'), Mage::helper('tax')->__('Manage Product Tax Classes'))
+            ->_addBreadcrumb(Mage::helper('tax')->__('Product Tax Classes'), Mage::helper('tax')->__('Product Tax Classes'))
         ;
         return $this;
     }
diff --git a/app/code/core/Mage/Adminhtml/controllers/Tax/RateController.php b/app/code/core/Mage/Adminhtml/controllers/Tax/RateController.php
index 299e39c893..87225225b7 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Tax/RateController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Tax/RateController.php
@@ -28,16 +28,16 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Zones and Rates'));
+             ->_title($this->__('Tax Zones and Rates'));
 
         /** @var Mage_Adminhtml_Block_Tax_Rate_Toolbar_Add $block */
         $block = $this->getLayout()->createBlock('adminhtml/tax_rate_toolbar_add', 'tax_rate_toolbar');
         $this->_initAction()
-            ->_addBreadcrumb(Mage::helper('tax')->__('Manage Tax Rates'), Mage::helper('tax')->__('Manage Tax Rates'))
+            ->_addBreadcrumb(Mage::helper('tax')->__('Tax Rates'), Mage::helper('tax')->__('Tax Rates'))
             ->_addContent(
                 $block
                     ->assign('createUrl', $this->getUrl('*/tax_rate/add'))
-                    ->assign('header', Mage::helper('tax')->__('Manage Tax Rates'))
+                    ->assign('header', Mage::helper('tax')->__('Tax Rates'))
             )
             ->_addContent($this->getLayout()->createBlock('adminhtml/tax_rate_grid', 'tax_rate_grid'))
             ->renderLayout();
@@ -52,7 +52,7 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
 
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Zones and Rates'));
+             ->_title($this->__('Tax Zones and Rates'));
 
         $this->_title($this->__('New Rate'));
 
@@ -65,7 +65,7 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
         /** @var Mage_Adminhtml_Block_Tax_Rate_Toolbar_Save $block */
         $block = $this->getLayout()->createBlock('adminhtml/tax_rate_toolbar_save');
         $this->_initAction()
-            ->_addBreadcrumb(Mage::helper('tax')->__('Manage Tax Rates'), Mage::helper('tax')->__('Manage Tax Rates'), $this->getUrl('*/tax_rate'))
+            ->_addBreadcrumb(Mage::helper('tax')->__('Tax Rates'), Mage::helper('tax')->__('Tax Rates'), $this->getUrl('*/tax_rate'))
             ->_addBreadcrumb(Mage::helper('tax')->__('New Tax Rate'), Mage::helper('tax')->__('New Tax Rate'))
             ->_addContent(
                 $block
@@ -121,7 +121,7 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Zones and Rates'));
+             ->_title($this->__('Tax Zones and Rates'));
 
         $rateId = (int)$this->getRequest()->getParam('rate');
         $rateModel = Mage::getSingleton('tax/calculation_rate')->load(null);
@@ -144,7 +144,7 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
         /** @var Mage_Adminhtml_Block_Tax_Rate_Toolbar_Save $block */
         $block = $this->getLayout()->createBlock('adminhtml/tax_rate_toolbar_save');
         $this->_initAction()
-            ->_addBreadcrumb(Mage::helper('tax')->__('Manage Tax Rates'), Mage::helper('tax')->__('Manage Tax Rates'), $this->getUrl('*/tax_rate'))
+            ->_addBreadcrumb(Mage::helper('tax')->__('Tax Rates'), Mage::helper('tax')->__('Tax Rates'), $this->getUrl('*/tax_rate'))
             ->_addBreadcrumb(Mage::helper('tax')->__('Edit Tax Rate'), Mage::helper('tax')->__('Edit Tax Rate'))
             ->_addContent(
                 $block
@@ -234,7 +234,7 @@ class Mage_Adminhtml_Tax_RateController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Zones and Rates'));
+             ->_title($this->__('Tax Zones and Rates'));
 
         $this->_title($this->__('Import and Export Tax Rates'));
 
diff --git a/app/code/core/Mage/Adminhtml/controllers/Tax/RuleController.php b/app/code/core/Mage/Adminhtml/controllers/Tax/RuleController.php
index 5362ec9663..8cd4f3ddf8 100644
--- a/app/code/core/Mage/Adminhtml/controllers/Tax/RuleController.php
+++ b/app/code/core/Mage/Adminhtml/controllers/Tax/RuleController.php
@@ -36,7 +36,7 @@ class Mage_Adminhtml_Tax_RuleController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Rules'));
+             ->_title($this->__('Tax Rules'));
 
         $this->_initAction()
             ->_addContent($this->getLayout()->createBlock('adminhtml/tax_rule'))
@@ -59,7 +59,7 @@ class Mage_Adminhtml_Tax_RuleController extends Mage_Adminhtml_Controller_Action
     {
         $this->_title($this->__('Sales'))
              ->_title($this->__('Tax'))
-             ->_title($this->__('Manage Tax Rules'));
+             ->_title($this->__('Tax Rules'));
 
         $taxRuleId  = $this->getRequest()->getParam('rule');
         $ruleModel  = Mage::getModel('tax/calculation_rule');
diff --git a/app/code/core/Mage/Adminhtml/etc/adminhtml.xml b/app/code/core/Mage/Adminhtml/etc/adminhtml.xml
index a48db6818a..637cee5216 100644
--- a/app/code/core/Mage/Adminhtml/etc/adminhtml.xml
+++ b/app/code/core/Mage/Adminhtml/etc/adminhtml.xml
@@ -57,7 +57,7 @@
                     </children>
                 </convert>
                 <currency translate="title">
-                    <title>Manage Currency Rates</title>
+                    <title>Currency Rates</title>
                     <action>adminhtml/system_currency</action>
                     <sort_order>50</sort_order>
                 </currency>
@@ -99,7 +99,7 @@
                     <sort_order>90</sort_order>
                 </cache>
                 <store translate="title" module="core">
-                    <title>Manage Stores</title>
+                    <title>Stores</title>
                     <action>adminhtml/system_store/</action>
                     <sort_order>100</sort_order>
                 </store>
@@ -148,7 +148,7 @@
                                 </children>
                             </acl>
                             <store translate="title">
-                                <title>Manage Stores</title>
+                                <title>Stores</title>
                             </store>
                             <design translate="title">
                                 <title>Design</title>
@@ -201,7 +201,7 @@
                                 </children>
                             </config>
                             <currency translate="title">
-                                <title>Manage Currency Rates</title>
+                                <title>Currency Rates</title>
                                 <sort_order>30</sort_order>
                             </currency>
                             <email_template translate="title">
diff --git a/app/code/core/Mage/Catalog/etc/adminhtml.xml b/app/code/core/Mage/Catalog/etc/adminhtml.xml
index c47c894911..01c12b2e13 100644
--- a/app/code/core/Mage/Catalog/etc/adminhtml.xml
+++ b/app/code/core/Mage/Catalog/etc/adminhtml.xml
@@ -25,12 +25,12 @@
             </depends>
             <children>
                 <products translate="title" module="catalog">
-                    <title>Manage Products</title>
+                    <title>Products</title>
                     <action>adminhtml/catalog_product/</action>
                     <sort_order>0</sort_order>
                 </products>
                 <categories translate="title" module="catalog">
-                    <title>Manage Categories</title>
+                    <title>Categories</title>
                     <action>adminhtml/catalog_category/</action>
                     <sort_order>10</sort_order>
                 </categories>
@@ -38,11 +38,11 @@
                     <title>Attributes</title>
                     <children>
                         <attributes translate="title" module="catalog">
-                            <title>Manage Attributes</title>
+                            <title>Attributes</title>
                             <action>adminhtml/catalog_product_attribute/</action>
                         </attributes>
                         <sets translate="title" module="catalog">
-                            <title>Manage Attribute Sets</title>
+                            <title>Attribute Sets</title>
                             <action>adminhtml/catalog_product_set/</action>
                         </sets>
                     </children>
@@ -79,18 +79,18 @@
                                 <title>Attributes</title>
                                 <children>
                                     <attributes translate="title">
-                                        <title>Manage Attributes</title>
+                                        <title>Attributes</title>
                                     </attributes>
                                     <sets translate="title">
-                                        <title>Manage Attribute Sets</title>
+                                        <title>Attribute Sets</title>
                                     </sets>
                                 </children>
                             </attributes>
                             <categories translate="title">
-                                <title>Manage Categories</title>
+                                <title>Categories</title>
                             </categories>
                             <products translate="title">
-                                <title>Manage Products</title>
+                                <title>Products</title>
                             </products>
                             <update_attributes translate="title">
                                 <title>Update Attributes</title>
diff --git a/app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php b/app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php
index 2ca6cb0687..24eec138c5 100644
--- a/app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php
+++ b/app/code/core/Mage/CurrencySymbol/Block/Adminhtml/System/Currencysymbol.php
@@ -45,7 +45,7 @@ class Mage_CurrencySymbol_Block_Adminhtml_System_Currencysymbol extends Mage_Adm
      */
     public function getHeader()
     {
-        return Mage::helper('adminhtml')->__('Manage Currency Symbols');
+        return Mage::helper('adminhtml')->__('Currency Symbols');
     }
 
     /**
diff --git a/app/code/core/Mage/CurrencySymbol/controllers/Adminhtml/System/CurrencysymbolController.php b/app/code/core/Mage/CurrencySymbol/controllers/Adminhtml/System/CurrencysymbolController.php
index 0812179670..824067c3da 100644
--- a/app/code/core/Mage/CurrencySymbol/controllers/Adminhtml/System/CurrencysymbolController.php
+++ b/app/code/core/Mage/CurrencySymbol/controllers/Adminhtml/System/CurrencysymbolController.php
@@ -40,12 +40,12 @@ class Mage_CurrencySymbol_Adminhtml_System_CurrencysymbolController extends Mage
                 Mage::helper('currencysymbol')->__('System')
             )
             ->_addBreadcrumb(
-                Mage::helper('currencysymbol')->__('Manage Currency Rates'),
-                Mage::helper('currencysymbol')->__('Manage Currency Rates')
+                Mage::helper('currencysymbol')->__('Currency Rates'),
+                Mage::helper('currencysymbol')->__('Currency Rates')
             );
 
         $this->_title($this->__('System'))
-            ->_title($this->__('Manage Currency Rates'));
+            ->_title($this->__('Currency Rates'));
         $this->renderLayout();
     }
 
diff --git a/app/code/core/Mage/CurrencySymbol/etc/adminhtml.xml b/app/code/core/Mage/CurrencySymbol/etc/adminhtml.xml
index 793097320b..01c918d0bb 100644
--- a/app/code/core/Mage/CurrencySymbol/etc/adminhtml.xml
+++ b/app/code/core/Mage/CurrencySymbol/etc/adminhtml.xml
@@ -19,7 +19,7 @@
         <system>
             <children>
                 <currency translate="title" module="currencysymbol">
-                    <title>Manage Currency</title>
+                    <title>Currency</title>
                     <children>
                         <rates translate="title">
                             <title>Rates</title>
@@ -44,7 +44,7 @@
                     <system>
                         <children>
                             <currency translate="title">
-                                <title>Manage Currency</title>
+                                <title>Currency</title>
                                 <children>
                                     <rates translate="title">
                                         <title>Rates</title>
diff --git a/app/code/core/Mage/Customer/etc/adminhtml.xml b/app/code/core/Mage/Customer/etc/adminhtml.xml
index 1369bc47fd..255da0c2e3 100644
--- a/app/code/core/Mage/Customer/etc/adminhtml.xml
+++ b/app/code/core/Mage/Customer/etc/adminhtml.xml
@@ -22,7 +22,7 @@
             <!-- action>adminhtml/customer/</action -->
             <children>
                 <manage translate="title" module="customer">
-                    <title>Manage Customers</title>
+                    <title>Customers</title>
                     <action>adminhtml/customer/</action>
                     <sort_order>0</sort_order>
                 </manage>
@@ -52,7 +52,7 @@
                                 <sort_order>10</sort_order>
                             </group>
                             <manage translate="title">
-                                <title>Manage Customers</title>
+                                <title>Customers</title>
                                 <sort_order>0</sort_order>
                             </manage>
                             <online translate="title">
diff --git a/app/code/core/Mage/Rating/etc/adminhtml.xml b/app/code/core/Mage/Rating/etc/adminhtml.xml
index 45075f0d44..78130207be 100644
--- a/app/code/core/Mage/Rating/etc/adminhtml.xml
+++ b/app/code/core/Mage/Rating/etc/adminhtml.xml
@@ -24,7 +24,7 @@
                             <reviews_ratings>
                                 <children>
                                     <ratings translate="title" module="rating">
-                                        <title>Manage Ratings</title>
+                                        <title>Ratings</title>
                                     </ratings>
                                  </children>
                             </reviews_ratings>
diff --git a/app/code/core/Mage/Review/etc/adminhtml.xml b/app/code/core/Mage/Review/etc/adminhtml.xml
index 36694941ba..76a29185e9 100644
--- a/app/code/core/Mage/Review/etc/adminhtml.xml
+++ b/app/code/core/Mage/Review/etc/adminhtml.xml
@@ -35,7 +35,7 @@
                             </children>
                         </reviews>
                         <ratings translate="title" module="review">
-                            <title>Manage Ratings</title>
+                            <title>Ratings</title>
                             <action>adminhtml/rating/index</action>
                         </ratings>
                     </children>
diff --git a/app/code/core/Mage/Tax/etc/adminhtml.xml b/app/code/core/Mage/Tax/etc/adminhtml.xml
index 593be22d25..f880ac9310 100644
--- a/app/code/core/Mage/Tax/etc/adminhtml.xml
+++ b/app/code/core/Mage/Tax/etc/adminhtml.xml
@@ -23,11 +23,11 @@
                     <sort_order>500</sort_order>
                     <children>
                         <rules translate="title" module="tax">
-                            <title>Manage Tax Rules</title>
+                            <title>Tax Rules</title>
                             <action>adminhtml/tax_rule</action>
                         </rules>
                         <rates translate="title" module="tax">
-                            <title>Manage Tax Zones &amp; Rates</title>
+                            <title>Tax Zones &amp; Rates</title>
                             <action>adminhtml/tax_rate</action>
                         </rates>
                         <import_export translate="title" module="tax">
@@ -70,11 +70,11 @@
                                         <sort_order>20</sort_order>
                                     </import_export>
                                     <rates translate="title">
-                                        <title>Manage Tax Zones &amp; Rates</title>
+                                        <title>Tax Zones &amp; Rates</title>
                                         <sort_order>30</sort_order>
                                     </rates>
                                     <rules translate="title">
-                                        <title>Manage Tax Rules</title>
+                                        <title>Tax Rules</title>
                                         <sort_order>40</sort_order>
                                     </rules>
                                 </children>
diff --git a/app/code/core/Mage/Widget/Block/Adminhtml/Widget/Instance.php b/app/code/core/Mage/Widget/Block/Adminhtml/Widget/Instance.php
index 2f7ad520e3..c4996f83a5 100644
--- a/app/code/core/Mage/Widget/Block/Adminhtml/Widget/Instance.php
+++ b/app/code/core/Mage/Widget/Block/Adminhtml/Widget/Instance.php
@@ -28,7 +28,7 @@ class Mage_Widget_Block_Adminhtml_Widget_Instance extends Mage_Adminhtml_Block_W
     {
         $this->_blockGroup = 'widget';
         $this->_controller = 'adminhtml_widget_instance';
-        $this->_headerText = Mage::helper('widget')->__('Manage Widget Instances');
+        $this->_headerText = Mage::helper('widget')->__('Widget Instances');
         parent::__construct();
         $this->_updateButton('add', 'label', Mage::helper('widget')->__('Add New Widget Instance'));
     }
diff --git a/app/code/core/Mage/Widget/controllers/Adminhtml/Widget/InstanceController.php b/app/code/core/Mage/Widget/controllers/Adminhtml/Widget/InstanceController.php
index 58a93a3a6f..1c601c6739 100644
--- a/app/code/core/Mage/Widget/controllers/Adminhtml/Widget/InstanceController.php
+++ b/app/code/core/Mage/Widget/controllers/Adminhtml/Widget/InstanceController.php
@@ -51,8 +51,8 @@ class Mage_Widget_Adminhtml_Widget_InstanceController extends Mage_Adminhtml_Con
                 Mage::helper('widget')->__('CMS')
             )
             ->_addBreadcrumb(
-                Mage::helper('widget')->__('Manage Widget Instances'),
-                Mage::helper('widget')->__('Manage Widget Instances')
+                Mage::helper('widget')->__('Widget Instances'),
+                Mage::helper('widget')->__('Widget Instances')
             );
         return $this;
     }
diff --git a/app/design/adminhtml/default/default/locale/en_US/translate.csv b/app/design/adminhtml/default/default/locale/en_US/translate.csv
index 0a0dc7b517..d70345f458 100644
--- a/app/design/adminhtml/default/default/locale/en_US/translate.csv
+++ b/app/design/adminhtml/default/default/locale/en_US/translate.csv
@@ -61,7 +61,7 @@
 "Option","Option"
 "Primary Billing Address","Primary Billing Address"
 "Create Backup","Create Backup"
-"Manage Options (values for your attribute)","Manage Options (values for your attribute)"
+"Options (values for your attribute)","Options (values for your attribute)"
 "Total Due","Total Due"
 "Copyright 2007","Copyright 2007"
 "Newsletter Subscribers","Newsletter Subscribers"
@@ -185,7 +185,7 @@
 "Add New Address","Add New Address"
 "Account Created in:","Account Created in:"
 "Default Billing","Default Billing"
-"Manage Titles (Size, Color, etc.)","Manage Titles (Size, Color, etc.)"
+"Titles (Size, Color, etc.)","Titles (Size, Color, etc.)"
 "Please confirm site switching. All data not saved will be lost.","Please confirm site switching. All data not saved will be lost."
 "Please select from each ratings above","Please select from each ratings above"
 "Qty Shipped","Qty Shipped"
diff --git a/app/design/adminhtml/default/default/template/catalog/product.phtml b/app/design/adminhtml/default/default/template/catalog/product.phtml
index 673c7b5970..bc3964d8b2 100644
--- a/app/design/adminhtml/default/default/template/catalog/product.phtml
+++ b/app/design/adminhtml/default/default/template/catalog/product.phtml
@@ -21,7 +21,7 @@
 <div class="content-header">
 <table cellspacing="0">
     <tr>
-        <td style="width:50%;"><h3 class="icon-head head-products"><?php echo Mage::helper('catalog')->__('Manage Products') ?></h3></td>
+        <td style="width:50%;"><h3 class="icon-head head-products"><?php echo Mage::helper('catalog')->__('Products') ?></h3></td>
         <td class="a-right">
             <?php echo $this->getButtonsHtml() ?>
         </td>
diff --git a/app/design/adminhtml/default/default/template/catalog/product/attribute/options.phtml b/app/design/adminhtml/default/default/template/catalog/product/attribute/options.phtml
index 8c64adeed0..e4d3182401 100644
--- a/app/design/adminhtml/default/default/template/catalog/product/attribute/options.phtml
+++ b/app/design/adminhtml/default/default/template/catalog/product/attribute/options.phtml
@@ -33,7 +33,7 @@
 
 <div class="entity-edit">
     <div class="entry-edit-head">
-        <h4 class="icon-head head-edit-form fieldset-legend"><?php echo Mage::helper('catalog')->__('Manage Titles (Size, Color, etc.)') ?></h4>
+        <h4 class="icon-head head-edit-form fieldset-legend"><?php echo Mage::helper('catalog')->__('Titles (Size, Color, etc.)') ?></h4>
     </div>
     <div class="box">
             <div class="hor-scroll">
@@ -58,7 +58,7 @@
 <br/>
 <div class="entity-edit" id="matage-options-panel">
     <div class="entry-edit-head">
-    <h4 class="icon-head head-edit-form fieldset-legend"><?php echo Mage::helper('catalog')->__('Manage Options (values of your attribute)') ?></h4>
+    <h4 class="icon-head head-edit-form fieldset-legend"><?php echo Mage::helper('catalog')->__('Options (values of your attribute)') ?></h4>
     </div>
     <div class="box">
         <div class="hor-scroll">
diff --git a/app/design/adminhtml/default/default/template/eav/attribute/options.phtml b/app/design/adminhtml/default/default/template/eav/attribute/options.phtml
index b1894fd71b..ed7b5c5498 100644
--- a/app/design/adminhtml/default/default/template/eav/attribute/options.phtml
+++ b/app/design/adminhtml/default/default/template/eav/attribute/options.phtml
@@ -33,7 +33,7 @@
 
 <div class="entity-edit">
     <div class="entry-edit-head">
-        <h4 class="icon-head head-edit-form fieldset-legend"><?php echo $this->__('Manage Titles (Size, Color, etc.)') ?></h4>
+        <h4 class="icon-head head-edit-form fieldset-legend"><?php echo $this->__('Titles (Size, Color, etc.)') ?></h4>
     </div>
     <div class="box">
             <div class="hor-scroll">
@@ -58,7 +58,7 @@
 <br/>
 <div class="entity-edit" id="matage-options-panel">
     <div class="entry-edit-head">
-    <h4 class="icon-head head-edit-form fieldset-legend"><?php echo $this->__('Manage Options (values of your attribute)') ?></h4>
+    <h4 class="icon-head head-edit-form fieldset-legend"><?php echo $this->__('Options (values of your attribute)') ?></h4>
     </div>
     <div class="box">
         <div class="hor-scroll">
diff --git a/app/design/adminhtml/default/default/template/system/config/switcher.phtml b/app/design/adminhtml/default/default/template/system/config/switcher.phtml
index b6d86d1d94..09e672dc3c 100644
--- a/app/design/adminhtml/default/default/template/system/config/switcher.phtml
+++ b/app/design/adminhtml/default/default/template/system/config/switcher.phtml
@@ -32,7 +32,7 @@
     </select>
     <?php if (Mage::getSingleton('admin/session')->isAllowed('system/store')): ?>
     <p style="margin:10px 0 0;">
-        <a href="<?php echo $this->getUrl('*/system_store') ?>"><?php echo $this->__('Manage Stores') ?></a>
+        <a href="<?php echo $this->getUrl('*/system_store') ?>"><?php echo $this->__('Stores') ?></a>
     </p>
     <?php endif ?>
 </div>
diff --git a/app/design/frontend/base/default/template/customer/account/dashboard/address.phtml b/app/design/frontend/base/default/template/customer/account/dashboard/address.phtml
index 0f2a9314b3..e8b9c5fc3a 100644
--- a/app/design/frontend/base/default/template/customer/account/dashboard/address.phtml
+++ b/app/design/frontend/base/default/template/customer/account/dashboard/address.phtml
@@ -17,7 +17,7 @@
     <div class="box">
         <div class="box-title">
             <h3><?php echo $this->__('Address Book') ?></h3>
-            <a href="<?php echo $this->getAddressBookUrl() ?>"><?php echo $this->__('Manage Addresses') ?></a>
+            <a href="<?php echo $this->getAddressBookUrl() ?>"><?php echo $this->__('Addresses') ?></a>
         </div>
         <div class="box-content">
             <div class="col-1">
diff --git a/app/design/frontend/rwd/default/template/customer/account/dashboard/address.phtml b/app/design/frontend/rwd/default/template/customer/account/dashboard/address.phtml
index 4d6d51762a..728cd0bafc 100644
--- a/app/design/frontend/rwd/default/template/customer/account/dashboard/address.phtml
+++ b/app/design/frontend/rwd/default/template/customer/account/dashboard/address.phtml
@@ -16,7 +16,7 @@
 <div class="box-account box-info">
     <div class="box-head">
         <h2><?php echo $this->__('Address Book') ?></h2>
-        <a href="<?php echo $this->getAddressBookUrl() ?>"><?php echo $this->__('Manage Addresses') ?></a>
+        <a href="<?php echo $this->getAddressBookUrl() ?>"><?php echo $this->__('Addresses') ?></a>
     </div>
     <div class="col2-set">
         <div class="col-1">
diff --git a/app/locale/en_US/Mage_Adminhtml.csv b/app/locale/en_US/Mage_Adminhtml.csv
index 61a65307dc..976c04cea0 100644
--- a/app/locale/en_US/Mage_Adminhtml.csv
+++ b/app/locale/en_US/Mage_Adminhtml.csv
@@ -571,18 +571,12 @@
 "Magento root directory","Magento root directory"
 "OpenMage %s","OpenMage %s"
 "Make sure that data encoding in the file is consistent and saved in one of supported encodings (UTF-8 or ANSI).","Make sure that data encoding in the file is consistent and saved in one of supported encodings (UTF-8 or ANSI)."
-"Manage Attribute Sets","Manage Attribute Sets"
-"Manage Attributes","Manage Attributes"
-"Manage Categories","Manage Categories"
-"Manage Content","Manage Content"
-"Manage Currency Rates","Manage Currency Rates"
-"Manage Customers","Manage Customers"
-"Manage Options (values of your attribute)","Manage Options (values of your attribute)"
-"Manage Ratings","Manage Ratings"
-"Manage Stores","Manage Stores"
-"Manage Tax Rules","Manage Tax Rules"
-"Manage Tax Zones and Rates","Manage Tax Zones and Rates"
-"Manage Titles (Size, Color, etc.)","Manage Titles (Size, Color, etc.)"
+"Content","Content"
+"Currency Rates","Currency Rates"
+"Options (values of your attribute)","Options (values of your attribute)"
+"Tax Rules","Tax Rules"
+"Tax Zones and Rates","Tax Zones and Rates"
+"Titles (Size, Color, etc.)","Titles (Size, Color, etc.)"
 "Manual","Manual"
 "Mass Product Assignment","Mass Product Assignment"
 "Matched Expression","Matched Expression"
diff --git a/app/locale/en_US/Mage_Catalog.csv b/app/locale/en_US/Mage_Catalog.csv
index 6f461c89f4..5df7f112cc 100644
--- a/app/locale/en_US/Mage_Catalog.csv
+++ b/app/locale/en_US/Mage_Catalog.csv
@@ -389,17 +389,12 @@
 "List Mode","List Mode"
 "List of Products that are set as New","List of Products that are set as New"
 "List of types","List of types"
-"Manage Attribute Sets","Manage Attribute Sets"
-"Manage Attributes","Manage Attributes"
-"Manage Catalog Categories","Manage Catalog Categories"
-"Manage Categories","Manage Categories"
-"Manage Label / Options","Manage Label / Options"
-"Manage Options (values of your attribute)","Manage Options (values of your attribute)"
-"Manage Product Attributes","Manage Product Attributes"
-"Manage Product Sets","Manage Product Sets"
-"Manage Products","Manage Products"
+"Catalog Categories","Catalog Categories"
+"Label / Options","Label / Options"
+"Options (values of your attribute)","Options (values of your attribute)"
+"Product Sets","Product Sets"
 "Manage Stock","Manage Stock"
-"Manage Titles (Size, Color, etc.)","Manage Titles (Size, Color, etc.)"
+"Titles (Size, Color, etc.)","Titles (Size, Color, etc.)"
 "Max Characters","Max Characters"
 "Maximal Depth","Maximal Depth"
 "Maximum Image Size","Maximum Image Size"
diff --git a/app/locale/en_US/Mage_Checkout.csv b/app/locale/en_US/Mage_Checkout.csv
index aece1f29fa..c9ad2ac293 100644
--- a/app/locale/en_US/Mage_Checkout.csv
+++ b/app/locale/en_US/Mage_Checkout.csv
@@ -161,7 +161,7 @@
 "Loading next step...","Loading next step..."
 "Login","Login"
 "Login/Registration Before","Login/Registration Before"
-"Manage Terms and Conditions","Manage Terms and Conditions"
+"Terms and Conditions","Terms and Conditions"
 "Maximum Display Recently Added Item(s)","Maximum Display Recently Added Item(s)"
 "Maximum qty allowed for Shipping to multiple addresses is %s","Maximum qty allowed for Shipping to multiple addresses is %s"
 "Mini-cart promotion block","Mini-cart promotion block"
diff --git a/app/locale/en_US/Mage_Cms.csv b/app/locale/en_US/Mage_Cms.csv
index 3920b54d59..9748335e68 100644
--- a/app/locale/en_US/Mage_Cms.csv
+++ b/app/locale/en_US/Mage_Cms.csv
@@ -75,7 +75,6 @@
 "Layout","Layout"
 "Layout Update XML","Layout Update XML"
 "Link to a CMS Page","Link to a CMS Page"
-"Manage Pages","Manage Pages"
 "Media Gallery","Media Gallery"
 "Media Storage","Media Storage"
 "Meta Data","Meta Data"
diff --git a/app/locale/en_US/Mage_Core.csv b/app/locale/en_US/Mage_Core.csv
index cee988a752..2c355dfe9f 100644
--- a/app/locale/en_US/Mage_Core.csv
+++ b/app/locale/en_US/Mage_Core.csv
@@ -205,7 +205,6 @@
 "Magento info API","Magento info API"
 "Mail Sending Settings","Mail Sending Settings"
 "Make sure that base URL ends with '/' (slash), e.g. http://yourdomain/magento/","Make sure that base URL ends with '/' (slash), e.g. http://yourdomain/magento/"
-"Manage Stores","Manage Stores"
 "Match expressions in the same order as displayed in the configuration.","Match expressions in the same order as displayed in the configuration."
 "Maximum Log Level","Maximum Log Level"
 "Media Storage","Media Storage"
diff --git a/app/locale/en_US/Mage_CurrencySymbol.csv b/app/locale/en_US/Mage_CurrencySymbol.csv
index c2702f3fea..98c82294b1 100644
--- a/app/locale/en_US/Mage_CurrencySymbol.csv
+++ b/app/locale/en_US/Mage_CurrencySymbol.csv
@@ -1,7 +1,6 @@
 "Currency","Currency"
 "Custom currency symbols were applied successfully.","Custom currency symbols were applied successfully."
-"Manage Currency","Manage Currency"
-"Manage Currency Rates","Manage Currency Rates"
+"Currency Rates","Currency Rates"
 "Rates","Rates"
 "Save Currency Symbols","Save Currency Symbols"
 "Symbol","Symbol"
diff --git a/app/locale/en_US/Mage_Customer.csv b/app/locale/en_US/Mage_Customer.csv
index e9c526331d..192c79bec5 100644
--- a/app/locale/en_US/Mage_Customer.csv
+++ b/app/locale/en_US/Mage_Customer.csv
@@ -212,8 +212,7 @@
 "Login and password are required.","Login and password are required."
 "Login or Create an Account","Login or Create an Account"
 "MM","MM"
-"Manage Addresses","Manage Addresses"
-"Manage Customers","Manage Customers"
+"Addresses","Addresses"
 "Maximum length must be less then %s symbols","Maximum length must be less than %s symbols"
 "Middle Name","Middle Name"
 "Minimum admin password length","Minimum admin password length"
diff --git a/app/locale/en_US/Mage_Rating.csv b/app/locale/en_US/Mage_Rating.csv
index 8c4df14444..20aecb9fe1 100644
--- a/app/locale/en_US/Mage_Rating.csv
+++ b/app/locale/en_US/Mage_Rating.csv
@@ -11,7 +11,6 @@
 "Edit Rating","Edit Rating"
 "ID","ID"
 "If you do not specify a rating title for a store, the default value will be used.","If you do not specify a rating title for a store, the default value will be used."
-"Manage Ratings","Manage Ratings"
 "New Rating","New Rating"
 "Option Label","Option Label"
 "Option Title","Option Title"
diff --git a/app/locale/en_US/Mage_Review.csv b/app/locale/en_US/Mage_Review.csv
index f72c05005b..e73e99acf4 100644
--- a/app/locale/en_US/Mage_Review.csv
+++ b/app/locale/en_US/Mage_Review.csv
@@ -45,7 +45,6 @@
 "How do you rate this product?","How do you rate this product?"
 "ID","ID"
 "Let us know your thoughts","Let us know your thoughts"
-"Manage Ratings","Manage Ratings"
 "My Product Reviews","My Product Reviews"
 "My Recent Reviews","My Recent Reviews"
 "Name","Name"
diff --git a/app/locale/en_US/Mage_SalesRule.csv b/app/locale/en_US/Mage_SalesRule.csv
index 151b165cc2..91e32f1100 100644
--- a/app/locale/en_US/Mage_SalesRule.csv
+++ b/app/locale/en_US/Mage_SalesRule.csv
@@ -62,7 +62,7 @@
 "Inactive","Inactive"
 "Item totals are not set for rule.","Item totals are not set for rule."
 "Labels","Labels"
-"Manage Coupon Codes","Manage Coupon Codes"
+"Coupon Codes","Coupon Codes"
 "Maximum Qty Discount is Applied To","Maximum Qty Discount is Applied To"
 "NOT FOUND","NOT FOUND"
 "NOT LOGGED IN","NOT LOGGED IN"
diff --git a/app/locale/en_US/Mage_Tag.csv b/app/locale/en_US/Mage_Tag.csv
index 3fe74078bd..3cc5fd682d 100644
--- a/app/locale/en_US/Mage_Tag.csv
+++ b/app/locale/en_US/Mage_Tag.csv
@@ -41,7 +41,6 @@
 "Info","Info"
 "Last Name","Last Name"
 "List","List"
-"Manage Tags","Manage Tags"
 "Middle Name","Middle Name"
 "My Recent Tags","My Recent Tags"
 "My Tags","My Tags"
diff --git a/app/locale/en_US/Mage_Tax.csv b/app/locale/en_US/Mage_Tax.csv
index 44cb3552f2..af03d65915 100644
--- a/app/locale/en_US/Mage_Tax.csv
+++ b/app/locale/en_US/Mage_Tax.csv
@@ -70,11 +70,11 @@
 "Including and Excluding Tax","Including and Excluding Tax"
 "Invalid file format upload attempt","Invalid file format upload attempt"
 "Invalid file upload attempt","Invalid file upload attempt"
-"Manage Customer Tax Classes","Manage Customer Tax Classes"
-"Manage Product Tax Classes","Manage Product Tax Classes"
-"Manage Tax Rates","Manage Tax Rates"
-"Manage Tax Rules","Manage Tax Rules"
-"Manage Tax Zones & Rates","Manage Tax Zones & Rates"
+"Customer Tax Classes","Customer Tax Classes"
+"Product Tax Classes","Product Tax Classes"
+"Tax Rates","Tax Rates"
+"Tax Rules","Tax Rules"
+"Tax Zones & Rates","Tax Zones & Rates"
 "Maximum zip code length is 9.","Maximum zip code length is 9."
 "Name","Name"
 "New Class","New Class"
diff --git a/app/locale/en_US/Mage_Widget.csv b/app/locale/en_US/Mage_Widget.csv
index c8395c5fe8..6d8af306fd 100644
--- a/app/locale/en_US/Mage_Widget.csv
+++ b/app/locale/en_US/Mage_Widget.csv
@@ -31,7 +31,7 @@
 "Insert Widget","Insert Widget"
 "Insert Widget...","Insert Widget..."
 "Layout Updates","Layout Updates"
-"Manage Widget Instances","Manage Widget Instances"
+"Widget Instances","Widget Instances"
 "New Instance","New Instance"
 "New Widget Instance","New Widget Instance"
 "Next page","Next page"
diff --git a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreEntityTest.php b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreEntityTest.php
index 7dfbb9642e..cfc93c740d 100644
--- a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreEntityTest.php
@@ -31,7 +31,7 @@ use Mage\Adminhtml\Test\Page\AdminLogout;
  *
  * Steps:
  * 1. Open Backend
- * 2. Go to System -> Manage Stores
+ * 2. Go to System -> Stores
  * 3. Click "Create Store View" button
  * 4. Fill data according to dataset
  * 5. Perform all assertions
diff --git a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreGroupEntityTest.php b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreGroupEntityTest.php
index a286c21a1e..9a92d37a2f 100644
--- a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreGroupEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateStoreGroupEntityTest.php
@@ -27,7 +27,7 @@ use Mage\Adminhtml\Test\Page\Adminhtml\DeleteStore;
 /**
  * Steps:
  * 1. Log in to backend.
- * 2. Go to System -> Manage Stores.
+ * 2. Go to System -> Stores.
  * 3. Click "Create Store" button.
  * 4. Fill data according to dataset.
  * 5. Click "Save Store" button.
diff --git a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateWebsiteEntityTest.php b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateWebsiteEntityTest.php
index ae013cd478..4228b32712 100644
--- a/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateWebsiteEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Adminhtml/Test/TestCase/CreateWebsiteEntityTest.php
@@ -29,7 +29,7 @@ use Mage\Adminhtml\Test\Page\Adminhtml\DeleteStore;
 /**
  * Steps:
  * 1. Open Backend.
- * 2. Go to System -> Manage Stores.
+ * 2. Go to System -> Stores.
  * 3. Click "Create Website" button.
  * 4. Fill data according to dataset.
  * 5. Create Store with created Website.
diff --git a/dev/tests/functional/tests/app/Mage/Bundle/Test/TestCase/CreateBundleProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Bundle/Test/TestCase/CreateBundleProductEntityTest.php
index cb21dca3b8..4ebe64c616 100644
--- a/dev/tests/functional/tests/app/Mage/Bundle/Test/TestCase/CreateBundleProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Bundle/Test/TestCase/CreateBundleProductEntityTest.php
@@ -24,7 +24,7 @@ use Magento\Mtf\TestCase\Injectable;
 /**
  * Steps:
  * 1. Login to the backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Click "Add Product".
  * 4. Select Bundle Product and Attribute set.
  * 5. Fill in all data according to data set.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateConfigurableProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateConfigurableProductEntityTest.php
index cf8d39780b..cb1e8ca953 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateConfigurableProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateConfigurableProductEntityTest.php
@@ -30,7 +30,7 @@ use Mage\Catalog\Test\Fixture\CatalogAttributeSet;
  *
  * Steps:
  * 1. Login to the backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Click "Add Product".
  * 4. Select Configurable Product and Attribute set.
  * 5. Select attribute.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateGroupedProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateGroupedProductEntityTest.php
index 5511b0a89a..eb4940a7aa 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateGroupedProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateGroupedProductEntityTest.php
@@ -25,7 +25,7 @@ use Mage\Catalog\Test\Page\Adminhtml\CatalogProduct;
 /**
  * Steps:
  * 1. Login to the backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Click "Add Product" button.
  * 4. Select "Grouped Product" in product type field and click "Continue".
  * 5. Fill in data according to attached data set.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateSimpleProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateSimpleProductEntityTest.php
index 10fa8aff6b..0712d0194d 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateSimpleProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateSimpleProductEntityTest.php
@@ -27,7 +27,7 @@ use Mage\Catalog\Test\Page\Adminhtml\CatalogProduct;
  *
  * Steps:
  * 1. Login to the backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Click "Add Product".
  * 4. Select "Simple Product" in product type field and click "Continue".
  * 5. Fill in data according to attached data set.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateVirtualProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateVirtualProductEntityTest.php
index 3de9e53aeb..39c397a802 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateVirtualProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/CreateVirtualProductEntityTest.php
@@ -26,7 +26,7 @@ use Magento\Mtf\TestCase\Injectable;
  *
  * Steps:
  * 1. Login to the backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Click "Add Product".
  * 4. Select "Virtual Product" in product type field and click "Continue".
  * 5. Fill in data according to attached data set.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/DuplicateProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/DuplicateProductEntityTest.php
index 0ac0eaf473..4f3cea6a02 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/DuplicateProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/DuplicateProductEntityTest.php
@@ -28,7 +28,7 @@ use Magento\Mtf\TestCase\Injectable;
  *
  * Steps:
  * 1. Login to backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Search and open product.
  * 4. Click "Duplicate".
  * 5. Perform asserts.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsCrossSellsTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsCrossSellsTest.php
index 96771f1e5d..052972f7d0 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsCrossSellsTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsCrossSellsTest.php
@@ -21,7 +21,7 @@ namespace Mage\Catalog\Test\TestCase\Product;
  *
  * Steps:
  * 1. Open Backend.
- * 2. Go to Catalog -> Manage Products.
+ * 2. Go to Catalog -> Products.
  * 3. Open config1 product.
  * 4. For config1 add as cross-sells: simp1, simp2.
  * 5. Save product.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsRelatedTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsRelatedTest.php
index 35750759f9..ee5f7e4a65 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsRelatedTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsRelatedTest.php
@@ -21,7 +21,7 @@ namespace Mage\Catalog\Test\TestCase\Product;
  *
  * Steps:
  * 1. Open Backend.
- * 2. Go to Catalog -> Manage Products.
+ * 2. Go to Catalog -> Products.
  * 3. Open simp1 product.
  * 4. For simp1 add as related: config1, simp2.
  * 5. Save product.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsUpSellsTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsUpSellsTest.php
index 8120fe565d..30a66e50cd 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsUpSellsTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/PromoteProductsAsUpSellsTest.php
@@ -21,7 +21,7 @@ namespace Mage\Catalog\Test\TestCase\Product;
  *
  * Steps:
  * 1. Open Backend.
- * 2. Go to Catalog -> Manage Products.
+ * 2. Go to Catalog -> Products.
  * 3. Open simp1 product.
  * 4. For simp1 add as up-sells: config1, simp2.
  * 5. Save product.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateConfigurableProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateConfigurableProductEntityTest.php
index f66b6693c4..c3e32e29d4 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateConfigurableProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateConfigurableProductEntityTest.php
@@ -26,7 +26,7 @@ use Magento\Mtf\TestCase\Scenario;
  *
  * Steps:
  * 1. Log in to backend.
- * 2. Open Catalog -> Manage Products.
+ * 2. Open Catalog -> Products.
  * 3. Search and open configurable product from preconditions.
  * 4. Fill in data according to dataset.
  * 5. Save product.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateSimpleProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateSimpleProductEntityTest.php
index 9cb282b2d8..dbd07cc0e6 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateSimpleProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/Product/UpdateSimpleProductEntityTest.php
@@ -28,7 +28,7 @@ use Magento\Mtf\TestCase\Injectable;
  *
  * Steps:
  * 1. Login to backend.
- * 2. Navigate to Catalog > Manage Products.
+ * 2. Navigate to Catalog > Products.
  * 3. Select a product in the grid.
  * 4. Edit test value(s) according to dataset.
  * 5. Click "Save".
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateAttributeSetEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateAttributeSetEntityTest.php
index 501c0df1f9..efd7dd1c82 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateAttributeSetEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateAttributeSetEntityTest.php
@@ -28,7 +28,7 @@ use Magento\Mtf\TestCase\Injectable;
  *
  * Steps:
  * 1. Log in to Backend.
- * 2. Navigate to Catalog > Attributes > Manage Attribute Sets.
+ * 2. Navigate to Catalog > Attributes > Attribute Sets.
  * 3. Start to create new Attribute Set.
  * 4. Fill out fields data according to data set.
  * 5. Add created Product Attribute to Attribute Set.
diff --git a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateProductAttributeEntityTest.php b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateProductAttributeEntityTest.php
index 7760138b1b..6eccde5a83 100644
--- a/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateProductAttributeEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Catalog/Test/TestCase/ProductAttribute/CreateProductAttributeEntityTest.php
@@ -25,7 +25,7 @@ use Mage\Catalog\Test\Fixture\CatalogAttributeSet;
  *
  * Test Flow:
  * 1. Log in to backend.
- * 2. Navigate to Catalog > Attributes > Manage Attributes.
+ * 2. Navigate to Catalog > Attributes > Attributes.
  * 3. Click 'Add New Attribute' button.
  * 4. Fill out fields data according to data set.
  * 5. Save Product Attribute.
diff --git a/dev/tests/functional/tests/app/Mage/CurrencySymbol/Test/TestCase/EditCurrencySymbolEntityTest.php b/dev/tests/functional/tests/app/Mage/CurrencySymbol/Test/TestCase/EditCurrencySymbolEntityTest.php
index 92e625b86a..c10463fd7b 100644
--- a/dev/tests/functional/tests/app/Mage/CurrencySymbol/Test/TestCase/EditCurrencySymbolEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/CurrencySymbol/Test/TestCase/EditCurrencySymbolEntityTest.php
@@ -30,12 +30,12 @@ use Mage\CurrencySymbol\Test\Page\Adminhtml\SystemCurrencySymbolIndex;
  * Test Flow:
  * Preconditions:
  * 1. Apply custom currency in Config.
- * 2. Navigate to Stores->Manage Currency->Rates and click to Import button.
+ * 2. Navigate to Stores->Currency->Rates and click to Import button.
  * 3. Create simple product.
  *
  * Steps:
  * 1. Login to backend.
- * 2. Navigate to Stores->Manage Currency->Symbols.
+ * 2. Navigate to Stores->Currency->Symbols.
  * 3. Make changes according to dataset.
  * 4. Click 'Save Currency Symbols' button.
  * 5. Perform all asserts.
diff --git a/dev/tests/functional/tests/app/Mage/Downloadable/Test/TestCase/CreateDownloadableProductEntityTest.php b/dev/tests/functional/tests/app/Mage/Downloadable/Test/TestCase/CreateDownloadableProductEntityTest.php
index f5da619e42..df7f916130 100644
--- a/dev/tests/functional/tests/app/Mage/Downloadable/Test/TestCase/CreateDownloadableProductEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Downloadable/Test/TestCase/CreateDownloadableProductEntityTest.php
@@ -24,7 +24,7 @@ use Magento\Mtf\TestCase\Injectable;
 /**
  * Steps:
  * 1. Log in to Backend.
- * 2. Navigate to Catalog -> Manage Products.
+ * 2. Navigate to Catalog -> Products.
  * 3. Start to create new Downloadable product.
  * 4. Fill in data according to data set.
  * 5. Save product.
diff --git a/dev/tests/functional/tests/app/Mage/Sales/Test/TestCase/CreateOrderFromBackendCustomerPageTest.php b/dev/tests/functional/tests/app/Mage/Sales/Test/TestCase/CreateOrderFromBackendCustomerPageTest.php
index 7444deb04c..6faa0acc3e 100644
--- a/dev/tests/functional/tests/app/Mage/Sales/Test/TestCase/CreateOrderFromBackendCustomerPageTest.php
+++ b/dev/tests/functional/tests/app/Mage/Sales/Test/TestCase/CreateOrderFromBackendCustomerPageTest.php
@@ -22,7 +22,7 @@ namespace Mage\Sales\Test\TestCase;
  *
  * Steps:
  * 1. Open Backend.
- * 2. Go to Customers -> Manage Customers.
+ * 2. Go to Customers -> Customers.
  * 3. Select customer created in preconditions.
  * 4. Click Create Order button.
  * 5. Add Products according to dataset.
diff --git a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRateEntityTest.php b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRateEntityTest.php
index a11e906ffc..216fc64999 100644
--- a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRateEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRateEntityTest.php
@@ -23,7 +23,7 @@ use Magento\Mtf\TestCase\Injectable;
 /**
  * Steps:
  * 1. Log in as default admin user.
- * 2. Go to Sales > Tax > Manage Tax Zones and Rates.
+ * 2. Go to Sales > Tax > Tax Zones and Rates.
  * 3. Click 'Add New Tax Rate' button.
  * 4. Fill in data according to dataset
  * 5. Save Tax Rate.
diff --git a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRuleEntityTest.php b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRuleEntityTest.php
index d821a653e8..5b2652616f 100644
--- a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRuleEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/CreateTaxRuleEntityTest.php
@@ -24,7 +24,7 @@ use Magento\Mtf\TestCase\Injectable;
  * Steps:
  *
  * 1. Log in to backend.
- * 2. Go to Sales -> Tax -> Manage Tax Rules.
+ * 2. Go to Sales -> Tax -> Tax Rules.
  * 3. Click 'Add New Tax Rule' button.
  * 4. Fill in data according to dataset.
  * 5. Save Tax Rule.
diff --git a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/DeleteTaxRateEntityTest.php b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/DeleteTaxRateEntityTest.php
index 7f80a00230..07c6b740f3 100644
--- a/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/DeleteTaxRateEntityTest.php
+++ b/dev/tests/functional/tests/app/Mage/Tax/Test/TestCase/DeleteTaxRateEntityTest.php
@@ -26,7 +26,7 @@ use Magento\Mtf\TestCase\Injectable;
  *
  * Steps:
  * 1. Log in as default admin user.
- * 2. Go to Sales -> Tax -> Manage Tax Zones and Rates.
+ * 2. Go to Sales -> Tax -> Tax Zones and Rates.
  * 3. Open created tax rate.
  * 4. Click "Delete Rate" button.
  * 5. Perform all assertions.
